### PR TITLE
Only future date optional prop added to DatePicker

### PR DESCRIPTION
**Pull request by adamalexandru4**
#173
Added new optional constraint for minimum date allowing only for future date or not.

### CODE REVIEW

**Comment by surdu** [comment]
Wouldn't be more flexible to have a `minDate` prop ?

File: src/components/form/datePicker.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ export const DatePicker = ({
   withTime,
   question,
   onAnswer,
-  currentResponse
+  currentResponse,
+  onlyFutureTime
```
Comment (surdu) [inline src/components/form/datePicker.js:16]: Wouldn't be more flexible to have it as a `minDate` prop instead ?

**Review by surdu — CHANGES_REQUESTED** [review]

File: src/components/form/datePicker.js [hunk 2]
```diff
@@ -57,7 +60,7 @@ export const DatePicker = ({
   );
 };
 
-const DateOnlyPicker = ({ startDate, maxDate, onChange }) => {
+const DateOnlyPicker = ({ startDate, maxDate, onChange, onlyFutureTime }) => {
```
Comment (surdu) [inline src/components/form/datePicker.js:63]: Wouldn't be more flexible to have it as a minDate prop instead ?

**Review by surdu — CHANGES_REQUESTED** [review]

File: src/components/form/datePicker.js (hunk 2 above)
Comment (adamalexandru4) [reply src/components/form/datePicker.js:63]: I used another field due to requirements. @aniri said that it should be a property set to true / false.
`minDate` should be a `Date` not a `Bool` by definition.

Comment (surdu) [reply src/components/form/datePicker.js:63]: I believe that she meant the [property in the question entry](https://github.com/code4romania/taskforce-fe-components/pull/171/files#diff-68772cae9a2fe16bd3cdefaa166253f5R15). This shouldn't be to different from what is already implemented for `allowFuture` in the previous link.

Comment (adamalexandru4) [reply src/components/form/datePicker.js:63]: Sorry for my mistake,. I will rework the code tonight.

Comment (aniri) [reply src/components/form/datePicker.js:63]: yes, @surdu is right, sorry for the confusion. as I mentioned in #173 , the implementation should be similar to the one for `allowFuture`

**Pull request closed without merging**